scripts/asm-installer/tests/common.sh

```diff
@@ -665,3 +670,55 @@ run_basic_test() {
 
   return "$SUCCESS"
 }
+
+create_service_account() {
+  WORKLOAD_SERVICE_ACCOUNT="vm-${LT_NAMESPACE}@${PROJECT_ID}.iam.gserviceaccount.com"
```

Comment:
Don't put the vm- prefix here, do it when you call the function. Also there's nothing specific to WORKLOAD here, just call this SERVICE_ACCOUNT (or SA)

Reply:
> Don't put the vm- prefix here, do it when you call the function. 
I don't understand the benefit here. I don't want to only add it in one place but forget to add it to other places.
> Also there's nothing specific to WORKLOAD here, just call this SERVICE_ACCOUNT (or SA)
This is very specific to workload. Renamed to workload_service_account.

Reply:
I mean that there's nothing specific to workloads with the gcloud command you're calling.

If you want to do it like this, I'd rather you have a generic create_service_account() function, and then you can wrap that with create_workload_service_account() where you add the vm- and whatever else. At some point we're going to have to test permissions and create service accounts, and whoever adds that logic is just going to have to pull the logic apart anyways.

I don't care enough to block the PR on it if you don't want to put it in this PR though.

Reply:
sgtm. I extract create_service_account out for now.